lib/src/resolveSymlinks.js

```diff
@@ -4,8 +4,10 @@ var through2 = require('through2');
 var fs = require('graceful-fs');
 var path = require('path');
 
-function resolveSymlinks() {
-  return through2.obj(resolveFile);
+function resolveSymlinks(followSymlinks) {
+  return followSymlinks ?
+    through2.obj(resolveFile) :
```

Comment:
i think instead of two functions, the resolveFile fn should take in the options object